src/TestFramework/Factory.php

```diff
@@ -85,6 +85,15 @@ public function __construct(
     }
 
     public function create(string $adapterName, bool $skipCoverage): TestFrameworkAdapter
+    {
+        $adapter = $this->constructAdapter($adapterName, $skipCoverage);
+
+        $adapter->checkVersion();
+
+        return $adapter;
+    }
+
+    private function constructAdapter(string $adapterName, bool $skipCoverage): TestFrameworkAdapter
```

Comment:
`construct`? `instantiate`? we need an alternative to `create` here

Reply:
.. `buildAdapter` / `initializeAdapter`

any option, just with `Adapter` postfix, please